features/unhandled_mach_exception.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Bugsnag captures an unhandled mach exception
+
+Scenario: Trigger a mach exception
+    When I run "UnhandledMachExceptionScenario" and relaunch the app
+    And I configure Bugsnag for "UnhandledMachExceptionScenario"
+    And I wait to receive a request
```

Comment:
is an assertion you could make that would confirm that this the expected mach exception, rather than some other error?

such as the error class matching `UnhandledMachExceptionScenario`?

Reply:
Updated with an extra assertion to confirm that the error message matches the scenario.